### PR TITLE
Added a stretch Positioned constructor.

### LINKED ISSUE #5835: FR: Positioned constructor that sets everything to 0.0 by default

### PR DESCRIPTION
**Pull request by dragostis**
Fixes #5835.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.fixedToEdge({
```
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/basic.dart]: This seems like a useful shorthand but maybe not the best name. How about Positioned.toEdges()?  Or you could define Positioned.fromInsets({ EdgeInsets insets }), like Positioned.fromRect(), and make the default for insets be EdgeInsets.zero.

Comment (dragostis) [inline packages/flutter/lib/src/widgets/basic.dart]: @Hixie What do you think about this one?

Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: I like fromInsets.

Lack of a good name is why I hadn't attempted this yet. :-)

Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: Another option is Positioned.cover()

Comment (dragostis) [inline packages/flutter/lib/src/widgets/basic.dart]: I like `Position.cover()` minus the fact that it doesn't describe the case when the positioned widget is below non-positioned widgets. How about `framed` or `enframed` or `constrained` or `tight`?

`fromInsets()` is a bit confusing to use without a parameter and `fromInsets(EdgeInsets.zero)` is a little too verbose.

Comment (HansMuller) [inline packages/flutter/lib/src/widgets/basic.dart]: You could provide Positioned.fromInsets() - without a default value for insets - and Positioned.cover() that was defined as Positioned.fromInsets with insets: EdgeInsets.zero

Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: cover refers to covering the stack, not covering the other things in the stack. I think it's better than the alternatives so far.
I'm fine with also having fromInsets.

Comment (dragostis) [inline packages/flutter/lib/src/widgets/basic.dart]: @Hixie I've updated the PR with `stretch()`. What do you think?

**Comment by dragostis** [comment]
I should also remember to check in any other uses that might have bee merged and rebase before actually merging.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.stretch({
+    Key key,
+    Widget child,
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: I'd still let you provide values for left, top, right, and bottom and just have 0.0 be the default for args you don't provide.

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/widgets/basic.dart (hunk 2 above)
Comment (dragostis) [inline packages/flutter/lib/src/widgets/basic.dart]: Fixed.

**Comment by Hixie** [comment]
Maybe 'fill' would be the better term, actually. https://docs.flutter.io/flutter/widgets/ImageFit-class.html

**Comment by abarth** [comment]
+1 for fill

**Pull request merged**